docs/Concepts/Qubernetes/Multicluster.md

```diff
@@ -0,0 +1,360 @@
+    ```bash
+    > qctl deploy network
+    ```
+
+7. **B** runs `qctl` to obtain the external nodes information from their running cluster.
+
+    ```bash
+    > qctl ls nodes --asexternal --node-ip=2.2.3.4
+    ```
+
+    ```yaml
+    external_nodes:
+    - Node_UserIdent:  quorum-node1
+      Enode_Url: "enode://dca1...@2.2.3.4:31923?discport=0&raftport=50401"
+      Tm_Url:  2.2.3.4:30482
+      Node_Acct_Addr: 0x9712A5aa48A654e9B117467748eF1F793153E918
+    - Node_UserIdent:  quorum-node2
+      Enode_Url: "enode://2a73...@2.2.3.4:32448?discport=0&raftport=50401"
+      Tm_Url:  2.2.3.4:30371
+      Node_Acct_Addr: 0xB4ba047619946B607dcBE6cA6E54Ff071f95c9E8
+    ```
+
+8. **B** shares its external nodes with **A** by copying and pasting the above output, and sending it
+to **A**.
+
+9. **A** adds the external nodes information from **B** to the **A** configuration file.
+
+    Updated config for **A**:
+
+    ```yaml
+    genesis:
+     consensus: istanbul
+     Quorum_Version: 2.7.0
+     Chain_Id: "1000"
+    nodes:
+    - Node_UserIdent: quorum-node1
+     Key_Dir: key1
+     quorum:
+       quorum:
+         consensus: istanbul
+         Quorum_Version: 2.7.0
+       tm:
+         Name: tessera
+         Tm_Version: 0.10.6
+     geth:
+       Geth_Startup_Params: ""
+    - Node_UserIdent: quorum-node2
+     Key_Dir: key2
+     quorum:
+       quorum:
+         consensus: istanbul
+         Quorum_Version: 2.7.0
+       tm:
+         Name: tessera
+         Tm_Version: 0.10.6
+     geth:
+       Geth_Startup_Params: ""
+    - Node_UserIdent: quorum-node3
+     Key_Dir: key3
+     quorum:
+       quorum:
+         consensus: istanbul
+         Quorum_Version: 2.7.0
+       tm:
+         Name: tessera
+         Tm_Version: 0.10.6
+     geth:
+       Geth_Startup_Params: ""
+    external_nodes:
+     - Node_UserIdent:  quorum-node1
+       Enode_Url: "enode://dca1...@2.2.3.4:31923?discport=0&raftport=50401"
+       Tm_Url:  2.2.3.4:30482
+       Node_Acct_Addr: 0x9712A5aa48A654e9B117467748eF1F793153E918
+     - Node_UserIdent:  quorum-node2
+       Enode_Url: "enode://2a73...@2.2.3.4:32448?discport=0&raftport=50401"
+       Tm_Url:  2.2.3.4:30371
+       Node_Acct_Addr: 0xB4ba047619946B607dcBE6cA6E54Ff071f95c9E8
+    ```
+
+10. **A** generates the update for the Kubernetes network resources. The update leaves the current
+nodes intact, only generating the diff and, updating the configuration files as necessary.
+
+    ```bash
+    > qctl generate network --update
+    ```
+
+11. **A** deploys the updated resources to its cluster.
+
+   ```bash
+   > qctl deploy network
+   ```
+
+   The clusters can now connect and there is a
+   GoQuorum Kubernetes network spanning multiple clusters.
+
+## Multi-cluster across namespaces using NodePorts
+
+A namespace in Kubernetes is an isolated cluster within the same running Kubernetes instance. We will walk through a
+flow of adding nodes across namespaces exposing the services via a NodePort service.
```

Comment:
Is the flow the following three paragraphs?  It's not quite clear to me.

Reply:
The "Multi-Cluster Across Namespaces Via NodePorts" section is the implementation / one way of representing external nodes, there will be others, e.g. K8s Ingress may "Multi-Cluster Across Namespaces Via NodePorts" and mentioning that other ways will be added in the future?

Reply:
Cool - thanks for the clarification